tests/unit/memory64/memory64_atomic_test.cc

```diff
@@ -56,6 +56,8 @@ class memory64_atomic_test_suite : public testing::TestWithParam<RunningMode>
         if (exec_env)
             wasm_runtime_destroy_exec_env(exec_env);
         if (module_inst)
+            wasm_runtime_deinstantiate(module_inst);
+        if (module)
             wasm_runtime_unload(module);
```

Comment:
i'm not sure if it's appropriate to unload module here.
isn't it simpler to make all clean up logic to TearDown?

Reply:
Yes, this resource management in some unit test suites is far from optimal. But I think I will address them in another PR focusing on that issue.